Recipely/Recipely/NetworkService/RecipeDetailResponseDTO.swift

```diff
@@ -0,0 +1,80 @@
+// RecipeDetailResponseDTO.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// RecipeDetailResponseDTO
+struct RecipeDetailResponseDTO: Codable {
+    /// Массив с подробными рецептами
+    let hits: [RecipeDetailHitDTO]
+}
+
+/// RecipeDetailHitDTO
+struct RecipeDetailHitDTO: Codable {
+    /// Подробный рецепт
+    let recipe: RecipeDetailDTO
+}
+
+/// Подробный рецепт
+struct RecipeDetailDTO: Codable {
+    /// Название рецепта
+    let label: String
+    /// Время приготовления
+    let totalTime: Double
+    /// Калории
+    let images: ImagesDTO
+    /// Масса рецепта
+    let totalWeight: Double
+    /// КБЖУ
+    let totalNutrients: TotalNutrientsDTO
+    /// Описание рецепта
+    let ingredientLines: [String]
+}
+
+/// ImagesDTO
+struct ImagesDTO: Codable {
```

Comment:
```suggestion
struct ImagesDTO: Codable {
```
говорил 1 модель 1 файл